lib/SIL/SILPrinter.cpp

```diff
@@ -2634,7 +2634,7 @@ void SILModule::print(SILPrintContext &PrintCtx, ModuleDecl *M,
   }
   
   OS << "\n\nimport Builtin\nimport " << STDLIB_NAME
```

Comment:
@kitasuke Let's make `BUILTIN_NAME` for `"Builtin"` constant as well. It's also used in AST.